### PR TITLE
i18n(es): Translated `build-forms-api.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `build-forms-api.mdx` to spanish.

### CODE REVIEW

File: src/content/docs/es/recipes/build-forms-api.mdx [hunk 1]
```diff
@@ -0,0 +1,401 @@
+---
+title: Construye formularios con rutas API
+description: Aprende como usar JavaScript para enviar formularios a una ruta API
+i18nReady: true
+type: recipe
+---
+import UIFrameworkTabs from "~/components/tabs/UIFrameworkTabs.astro";
+import PackageManagerTabs from "~/components/tabs/PackageManagerTabs.astro"
+
+Un formulario HTML hace que el navegador se actualice o navegue a una nueva página. Para enviar datos de formulario a un punto final de API en su lugar, debe interceptar el envío del formulario usando JavaScript.
```
Comment (Waxer59) [inline src/content/docs/es/recipes/build-forms-api.mdx:10]:
```suggestion
Un formulario HTML hace que el navegador se actualice o navegue a una nueva página. Para enviar datos de formulario a un endpoint de API en su lugar, debes interceptar el envío del formulario usando JavaScript.
```

File: src/content/docs/es/recipes/build-forms-api.mdx [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+
+Esta receta muestra como enviar datos de un formulario a un punto final de API y manejar esos datos.
```
Comment (Waxer59) [inline src/content/docs/es/recipes/build-forms-api.mdx:12]:
```suggestion
Esta receta muestra como enviar datos de un formulario a un endpoint de API y manejar esos datos.
```

File: src/content/docs/es/recipes/build-forms-api.mdx [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+            message: "Success!"
+          }),
+          { status: 200 }
+        );
+      };
+      ```
+
+  3. Crea una función que acepte un evento de envío, luego pásalo como un manejador de eventos `submit` a tu formulario. En la función, llama a `preventDefault` en el evento para anular el proceso de envío predeterminado del navegador.
+  
+      Luego, crea un objeto `FormData` y envíalo a tu punto final usando `fetch`.
```
Comment (Waxer59) [inline src/content/docs/es/recipes/build-forms-api.mdx:169]:
```suggestion
      Luego, crea un objeto `FormData` y envíalo a tu endpoint usando `fetch`.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias

**Comment by dreyfus92** [comment]
Gracias por la revisión @Waxer59

**Review by Waxer59 — APPROVED** [review]
LGTM ✔

**Review by Yan-Thomas — APPROVED** [review]
LGTM! 🚀

**Pull request merged**